e2e_tests/tests/command/test_run.py

```diff
@@ -201,6 +202,28 @@ def test_large_uploads(tmp_path: Path) -> None:
             ]
         )
 
+    with FileTree(tmp_path, {"hello.py": "print('hello world')"}) as tree:
```

Comment:
may as well just test this with integrations, honestly. i dont see that this test adds a lot if the APIs are tested and the experiment test already exists,

Reply:
I do like this test, (I'll remove the other part from e2e_cpu and just have it run in e2e_k8s) as a smoke test

We have had a bunch of people report this issue so I think it makes sense to know we support it forever with this user perspective test. I guess its really not likely it will fail based on the how system works.

Reply:
What integration tests are you thinking to add? 

could probaly test command's prestart but that is maybe only slightly annoying with actors.

Reply:
Added a command integration test, making the large context test only run on k8s tests (once per commit merged) rather than on every e2e_test